File: function.cs
// function.cs
using System;

class Program
{
    static int MyFunction(int param) {
        int x = param;
        x += 10;
        if (x > 15) {
            x -= 5;
        } else {
            x += 5;
        }
        for (int i = 0; i < 3; i++) {
            x += i;
        }
        return x;
    }

    static void Main() {
        Console.Write("Enter an integer: ");
        string input = Console.ReadLine();
        int param;
        if (int.TryParse(input, out param)) {
            Console.WriteLine(MyFunction(param));
        } else {
            Console.WriteLine("Invalid input");
        }
    }
}

File: function.pas
{ function.pas }
program FunctionExample;

function MyFunction(Param: Integer): Integer;
var
    x, i: Integer;
begin
    x := Param + 10;
    if x > 15 then
        x := x - 5
    else
        x := x + 5;
    for i := 0 to 2 do
        x := x + i;
    MyFunction := x;
end;

var
    input: string;
    param: Integer;

begin
    Write('Enter an integer: ');
    Readln(input);
    Val(input, param);
    if (param = 0) and (input <> '0') then
        Writeln('Invalid input')
    else
        Writeln(MyFunction(param));
end.
